src/HypothesisClient/Model/Annotation.php

```diff
@@ -22,26 +22,16 @@
 
     /**
      * @internal
-     *
-     * @param string            $id
-     * @param string|null       $text
-     * @param DateTimeImmutable $created
-     * @param DateTimeImmutable $updated
-     * @param Document          $document
-     * @param Target            $target
-     * @param string            $uri
-     * @param array|null        $references
-     * @param $permissions
      */
     public function __construct(
         string $id,
-        $text,
+        string $text = null,
         DateTimeImmutable $created,
         DateTimeImmutable $updated,
         Document $document,
         Target $target,
         string $uri,
-        $references,
+        array $references,
```

Comment:
this was nullable in the API (missing field), so I'll check in the Serializer how that value is dealt with